### ISSUE TITLE
Encoding::CompatibilityError: incompatible character encodings: UTF-8 and ASCII-8BIT

### ISSUE THREAD

**Issue by ioveracker** [danger/danger#409]
I updated from v0.7.4 to v2.1.1 today and this error popped up when running `danger local`. It goes away if I downgrade to v0.10.1, but appears as soon as I update to 2.0 and up.

```
❯❯❯ bundle exec danger local
Running your Dangerfile against this PR - https://github.com/.../pull/467
Turning on --verbose

Info:

Encoding::CompatibilityError: incompatible character encodings: UTF-8 and ASCII-8BIT
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:120:in `join'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:120:in `render'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cork-0.1.0/lib/cork/board.rb:67:in `puts'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cork-0.1.0/lib/cork/board.rb:67:in `puts'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/danger-2.1.1/lib/danger/danger_core/dangerfile.rb:159:in `block in print_known_info'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/cork-0.1.0/lib/cork/board.rb:184:in `section'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/danger-2.1.1/lib/danger/danger_core/dangerfile.rb:157:in `print_known_info'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/danger-2.1.1/lib/danger/danger_core/dangerfile.rb:167:in `parse'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/danger-2.1.1/lib/danger/commands/local.rb:109:in `run'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
  /Users/ioveracker/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/danger-2.1.1/bin/danger:5:in `<top (required)>'
  /Users/ioveracker/.rbenv/versions/2.2.3/bin/danger:23:in `load'
  /Users/ioveracker/.rbenv/versions/2.2.3/bin/danger:23:in `<top (required)>'
```

**Comment by orta** [comment]
Hrm, I have no idea about these kind of problems. Are you passing emoji or interesting characters that can help in figuring the debugging?

**Comment by ioveracker** [comment]
Yeah, the PR that it was running against does have the 📺 emoji in the title.

**Comment by orta** [comment]
Cool, that would have got shown in the table, wonder if `terminal-table` only does ASCII, so we'd have to strip it down

**Comment by orta** [comment]
Yep, looks like there's a gem fork just for unicode - https://github.com/rochefort/terminal-table-unicode

**Comment by orta** [comment]
Hrm, that adds a bunch of new dependencies, I'd rather strip unicode I'm afraid.

**Comment by orta** [comment]
Looks like the [answer is string encode](http://stackoverflow.com/questions/1268289/how-to-get-rid-of-non-ascii-characters-in-ruby)

**Comment by ioveracker** [comment]
Yeah, stripping would certainly be acceptable for the `danger local` terminal output. I wonder why it works in versions prior to Danger 2.0, though?

**Comment by orta** [comment]
[Cork](https://github.com/CocoaPods/Cork/) was introduced as a UI dependency injection tool [sometime in-between](https://rubygems.org/gems/danger/versions/0.7.1), bet it's somewhere in that.

**Comment by orta** [comment]
Should be fixed by #625

**Comment by JuanitoFatas** [comment]
Please try v3.5.3, thanks!
